src/js/webSerial.js

```diff
@@ -234,7 +236,7 @@ class WebSerial extends EventTarget {
             );
         }
         return {
-            bytesSent: this.bytesSent,
+            bytesSent: data.byteLength,
```

Comment:
Think this should stay to distinguish from `bytesReceived`.

Reply:
The problem is that we have some part in the msp code that verifies that this method returns the same size that sent. I will search the code later, now I'm on mobile. 
We can change it here, but we will need to adapt the other code to not verify that.

Reply:
This is the line I was talking: https://github.com/betaflight/betaflight-configurator/blob/d465f264bec291a145cbc57b69d47302f787a3e4/src/js/msp.js#L360